director/model/terminal.py

```diff
@@ -21,6 +21,13 @@ class Terminal(db.Model):
     lab_id = db.Column(db.Integer, ForeignKey('lab.id'))
     sessions = db.relationship("Session", backref="terminal")
 
+    def create_terminal():
```

Comment:
There is no need for a method like this here.

Reply:
removing it to